src/docs/integrations/slack/index.mdx

```diff
@@ -75,16 +88,23 @@ You’ll see “Verified” when you’ve entered the correct URL.
 
 _Note: When you enter the Request URL, Slack tries to verify it. The request will fail if you didn’t configure the client keys/secret and restarted Sentry to ensure those config changes were picked up. If it fails to validate, first make sure your Sentry instance is running and Slack can talk to it (Slack [doesn’t have an ip range](https://twitter.com/slackapi/status/567110311476350976?lang=en))_
```

Comment:
This should probably be an `<Alert level="warning">`. See https://github.com/getsentry/develop/pull/266/files#diff-3b597fb418bf701a73c54098e70e48766f279625028f271d7415a741c0ab6e64R15 for an example.